fix "undefined"

EnderStudiosDEV opened the pull request.

EnderStudiosDEV committed:
fix "undefined"

BamBoozledMC commented:
Please update your index.js file and re add the changes, your index.js file is out dated (check if user has role was reverted)

SnowedSeal commented:
okay

EnderStudiosDEV committed:
fixes

fixed lurklist issue
and updated to latest sealbot vers (hopefully it works this time)

EnderStudiosDEV committed:
fixed undefined and removed from lurk on logout
